config/crd/bases/skupper_listener_crd.yaml

```
@@ -40,11 +75,24 @@ spec:
               type: object
               properties:
                 status:
+                  description: |-
+                    The current state of the resource.
+                    `Pending`: The resource is being processed.
```

```suggestion
                    - `Pending`: The resource is being processed.
```
repeat for other items to improve rendering